incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -165,17 +166,45 @@ func TestNonRBACTypes(t *testing.T) {
 
 func TestPropagateConflict(t *testing.T) {
 	tests := []struct {
-		name         string
-		inNamespaces string
-		allow        bool
+		name            string
+		forest          string
+		inNsesWithAnnot map[string]map[string]string
```

Comment:
Will you undestand what this means in six months? Options to fix include:
* Comments
* Self-documenting types

Reply:
Added comment `inNsesWithAnnot contains a map of the namespaces we are creating the objects in, and the values are the annotations for the objects in each namespace`